### PR TITLE
Correct computed value of transform

### PR DESCRIPTION
**Pull request by csnardi**
Per https://drafts.csswg.org/css-transforms-1/#serialization-of-the-computed-value, the computed value of a `<transform-list>` is one `matrix()`. Change this test to bring it in line with the spec, matching the behavior of all major browsers.

### CODE REVIEW

**Comment by csnardi** [comment]
@ewilligers Can you review? I believe I read the spec correctly here.

**Review by ewilligers — APPROVED** [review]

**Comment by sideshowbarker** [comment]
@ewilligers @frivoal is this PR waiting on anything? review from @smfr?

**Pull request merged**
